packages/ethers/src/index.ts

```diff
@@ -91,17 +96,14 @@ export class TurnkeySigner extends ethers.Signer {
   private async _signTransactionImpl(message: string): Promise<string> {
     const { activity } = await PublicApiService.postSignTransaction({
       body: {
-        request: {
-          requestId: process.hrtime().join(""), // nanosecond timestamp
-          organizationId: this.config.organizationId,
-          type: "ACTIVITY_TYPE_SIGN_TRANSACTION",
-          intent: {
-            keyId: this.config.keyId,
-            organizationId: this.config.organizationId,
-            type: "TRANSACTION_TYPE_ETHEREUM",
-            unsignedTransaction: message,
-          },
+        type: "ACTIVITY_TYPE_SIGN_TRANSACTION",
+        organizationId: this.config.organizationId,
+        parameters: {
+          keyId: this.config.privateKeyId,
```

Comment:
hm.... should we update the shape of `ACTIVITY_TYPE_SIGN_TRANSACTION` requests to take in a `privateKeyId` parameter? 🤔

Reply:
yeah we can, the ship hasn't fully sailed yet -- I'll bump another version if we end up making that change

Reply:
I think it would be nice for consistency's sake -- that way there's no confusion if the changelog says we replaced `keyId` with `privateKeyId`, yet we still have `keyId` around